src/status_im/contexts/communities/overview/style.cljs

```diff
@@ -1,42 +1,142 @@
 (ns status-im.contexts.communities.overview.style
   (:require
-    [status-im.contexts.shell.constants :as shell.constants]))
-
-(def screen-horizontal-padding 20)
-
-(def last-community-tag
-  {:margin-right (* 2 screen-horizontal-padding)})
-
-(def community-tag-container
-  {:padding-horizontal screen-horizontal-padding
-   :margin-horizontal  (- screen-horizontal-padding)
-   :margin-bottom      16})
-
-(def community-content-container
-  {:padding-horizontal screen-horizontal-padding})
+    [quo.foundations.colors :as colors]
+    [react-native.core :as rn]
+    [react-native.safe-area :as safe-area]))
 
 (defn fetching-placeholder
   [top-inset]
   {:flex       1
    :margin-top top-inset})
 
-(def blur-channel-header
-  {:position :absolute
-   :top      100
-   :height   34
-   :right    0
-   :left     0
-   :flex     1})
-
 (def community-overview-container
-  {:position :absolute
-   :top      0
-   :left     0
-   :right    0
-   :bottom   0})
-
-(defn channel-list-component
-  []
-  {:margin-top    8
-   :margin-bottom (+ 21 shell.constants/floating-shell-button-height)
-   :flex          1})
+  {:flex 1})
+
+(defn header-cover-image
+  [background-color]
+  {:width            "100%"
```

Comment:
could you pls elaborate on why "100%" is needed

Reply:
Sure! @flexsurfer 
Since I'm not providing `flex 1` (I'm giving a fixed height), I preferred to just ask the container to use all of its width.

Reply:
Could we also use `flex: 1` combine with a `flex-direction: row` ?

Reply:
Yep, we could @seanstrom, but it works this way, I usually when dealing with images we need to provide a more specific fixed size, instead of a dynamic one (flex). It works this way, I'd keep it as is. wdyt?